docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+version: '2.4'
+
+services:
+
+  docker1.webhook.healthchain.cr:
+    image: soinsalud/dockerhub-slack-webhook:latest
```

Comment:
Why do we need to use soinsalud's dockerhub image?

And `container_name` and `hostname` are needless.

I prefer to followings

```yaml
  app:
    build: .
    working_dir: "/app"
```